cli/azd/test/functional/testdata/samples/webapp/src/dotnet/webapp.csproj

```diff
@@ -1,9 +1,10 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
+    <UserSecretsId>995405db-16c7-489f-8f57-3b1983905753</UserSecretsId>
```

Comment:
Intentional change?

Reply:
No - This constantly trips me up - I should stop using the template from within the source tree like this.  Thanks for pointing it out.